public/data/Kendall_Does_Coding.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "Kendall Pinto",
+  "type": "personal",
+  "bio": "Hello. My name is Kendall. I am a Software Developer / Coder. I have made a bunch of programs in various languages, like Python, Javascript, and the one I'm using to make this website, HTML & CSS.",
+  "avatar": "https://i.ibb.co/5LsjwVB/Avatar-Maker.png",
+  "links": [
+    {
+      "name": "Check out my personal website",
+      "url": "https://kendalldoescoding.gq/",
+      "icon": "globe"
+    },
+    {
+      "name": "Take a look at my GitHub",
+      "url": "https://github.com/kendalldoescoding",
+      "icon": "github"
+    },
+    {
+      "name": "Take a look at my GitLab",
+      "url": "https://gitlab.com/kendalldoescoding",
+      "icon": "https://about.gitlab.com/images/press/press-kit-icon.svg"
+    },
+    {
+      "name": "Quiz",
+      "url": "https://kendalldoescoding.gq/quiz",
+      "icon": "https://cdn-icons.flaticon.com/png/512/3261/premium/3261259.png?token=exp=1643398341~hmac=30c8bd086994bd1a76b2810fefdf0b4a"
```

Comment:
Please choose from the available icons in the readme

Reply:
Or, I can use icon: link?